BUILDING.md

```diff
@@ -235,7 +235,7 @@ The Node.js project supports Python >= 3 for building and testing.
 
 * `gcc` and `g++` >= 8.3 or newer, or
 * GNU Make 3.81 or newer
-* Python 3.6, 3.7, 3.8, or 3.9 (see note above)
+* Python 3.6, 3.7, 3.8, 3.9 or 3.10 (see note above)
```

Comment:
```suggestion
* Python 3.6, 3.7, 3.8, 3.9, or 3.10 (see note above)
```

Reply:
Since this list is getting pretty long maybe we could just specify a range instead of listing each individual version?

Reply:
I think 5 versions of the list listing are still acceptable.